packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/internal/AppConfigurationImpl.kt

```diff
@@ -107,23 +105,34 @@ public class AppConfigurationImpl constructor(
     private fun initializeSyncClientConfig(): RealmSyncClientConfigurationPointer =
         RealmInterop.realm_sync_client_config_new()
             .also { syncClientConfig ->
-                // TODO use separate logger for sync or piggyback on config's?
```

Comment:
This change will make the sync logs use the logger defined by the App, which is needed to use the same custom loggers and log-level, but it also prevents us from changing the tag. Before this change, the sync logs used a SYNC tag and App used REALM. After this change, both use REALM. I'm not sure how big of a problem this, but we probably need to sit down and take a hard look at how the log API is designed and distributed across all of our components. Especially in the light of a unified logging framework in the works in Core.

Reply:
I am not really sure that I get the full details of this ... but do we need to note this in the CHANGELOG?